### ISSUE TITLE
Unable to build?

### ISSUE THREAD

**Issue by bitvijays** [adaptant-labs/k8s-dt-node-labeller#1]
Hello Everyone,

Hope you are doing well. Thank you for working on this. I did tried to build the program using 
```
go build
```
and got the error
```
../go/pkg/mod/k8s.io/client-go@v0.17.2/tools/clientcmd/validation.go:73:5: cannot use errConfigurationInvalid literal (type errConfigurationInvalid) as type "k8s.io/apimachinery/pkg/util/errors".Aggregate in assignment:
        errConfigurationInvalid does not implement "k8s.io/apimachinery/pkg/util/errors".Aggregate (missing Is method)
```

Suggestions for building this properly?

**Comment by pmundt** [comment]
I ran into the same issue refreshing the dependencies on a clean checkout - it seems to have been caused by some of the K8s deps shifting to a newer API version (0.18.0) while others remained at an earlier version (0.17.2). This should now be fixed in 07e09f5f74705c1f0bf109aa379fe0c25253c510.

**Comment by bitvijays** [comment]
Thank you @pmundt for fixing this :)
